packages/core/src/components/dropdown/_dropdown.scss

```diff
@@ -0,0 +1,364 @@
+      outline: 0;
+      border: 0;
+      cursor: pointer;
+      color: rgba(0, 0, 0, 0);
+
+      &::-ms-expand {
+        display: none;
+      }
+
+      [dir='rtl'] & {
+        padding: 0 $ray-field-h-spacing 0 1.75rem;
+      }
+    }
+
+    &__body + &__input {
+      visibility: collapse;
+      pointer-events: none;
+      width: 0;
+      height: 0;
+      padding: 0;
+    }
+
+    &__body {
+      display: flex;
+      align-items: center;
+    }
+
+    &__selected {
+      flex: 0 1 auto;
+      max-width: 100%;
+      &-value {
+        display: block;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        line-height: $dropdown-clear-btn-size;
+      }
+      &-clear {
+        display: block;
+        text-align: center;
+        visibility: collapse;
+        color: $ray-color-text-medium;
+        line-height: $dropdown-clear-btn-size;
+        width: $dropdown-clear-btn-size;
+        height: $dropdown-clear-btn-size;
+      }
+    }
+
+    &__option {
+      $list-item-gap: 0.2rem;
+      padding: 0 1.75rem 0 $ray-field-h-spacing;
+      height: $dropdown-option-size;
+      display: flex;
+      align-items: center;
+      margin: $list-item-gap;
+      cursor: pointer;
+      &[disabled] {
+        pointer-events: none;
+        color: rgba($ray-color-gray-60, 0.4);
+      }
+      &:hover {
+        background-color: rgba($ray-color-blue-10, .5);
+        box-shadow: 0 0 0 $list-item-gap rgba($ray-color-blue-10, .5);
+      }
+      &:focus {
+        outline: 0;
+        box-shadow: $ray-box-shadow-focus-state;
+        cursor: default;
+      }
+      &--separator {
+        height: 1px;
+        margin: 0;
+        background-color: $ray-color-gray-10;
+      }
+      &--selected {
+        color: $ray-color-blue-50;
+      }
+      &--group-label {
+        text-indent: $ray-field-h-spacing + $list-item-gap;
+        pointer-events: none;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        display: block;
+        overflow: hidden;
+        color: $ray-color-gray-60;
+        height: $dropdown-option-size;
+        line-height: $dropdown-option-size;
+        border-bottom: 1px solid $ray-color-gray-10;
+        & ~ .#{$ray-class-prefix}#{$component-name}__option {
+          text-indent: $ray-field-h-spacing;
+        }
+      }
+      &-container {
+        position: absolute;
+        visibility: hidden;
+        z-index: 4;
+        top: 100%;
+        left: -1px;
+        right: -1px;
+        height: 0;
+        background-color: $ray-color-white;
+        border: $ray-border-width solid $ray-color-gray-60;
+        border-top: 0;
+        border-radius: 0 0 $ray-border-radius $ray-border-radius;
+        max-height: $dropdown-max-height;
+        overflow-y: auto;
+        transition-duration: 0.25s;
+        transition-timing-function: ease-in;
+        transition-property: height, max-height;
+      }
+      &-list {
```

Comment:
delete unused selector

Reply:
done

Reply:
There's still Coverage decreased (-4.5%) , could you add more tests? You can view the coveralls report to see which lines need coverage

Reply:
added more tests